Makefile

```diff
@@ -294,7 +294,22 @@ ifeq ($(XENOMAI_VERSION),3)
   BELA_USE_DEFINE=BELA_USE_RTDM
 endif
 
-DEFAULT_COMMON_FLAGS := $(DEFAULT_XENOMAI_CFLAGS) -O3 -g -march=armv7-a -mtune=cortex-a8 -mfloat-abi=hard -mfpu=neon -ftree-vectorize -ffast-math -DNDEBUG -D$(BELA_USE_DEFINE) -I$(BASE_DIR)/resources/$(DEBIAN_VERSION)/include -save-temps=obj
+IS_AM572x = $(shell grep -q AI /proc/device-tree/model && echo '-DIS_AM572x' || echo ' ')
```

Comment:
you shouldn't need the `|| echo ' '` because when using the variable you will always surround it with whitespaces.

Reply:
> when using the variable you will always surround it with whitespaces

Oh I see.. then I will remove the `|| ...` part.